cmd/finch/virtual_machine_remove_test.go

```diff
@@ -49,19 +52,28 @@ func TestRemoveVMAction_runAdapter(t *testing.T) {
 				logger.EXPECT().Info(gomock.Any()).AnyTimes()
 			},
 		},
+		{
+			name: "should forcibly remove the instance",
+			cmd:  removeCmd,
+			args: []string{
+				"--force",
+			},
+			mockSvc: func(logger *mocks.Logger, creator *mocks.LimaCmdCreator, ctrl *gomock.Controller) {
+				command := mocks.NewCommand(ctrl)
+				creator.EXPECT().CreateWithoutStdio("remove", "--force", limaInstanceName).Return(command)
+				command.EXPECT().CombinedOutput()
+				logger.EXPECT().Info(gomock.Any()).AnyTimes()
+			},
+		},
 	}
 
+	//nolint:paralleltest // run in serial because tests with different args can interfere with each other
 	for _, tc := range testCases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			t.Parallel()
-
-			ctrl := gomock.NewController(t)
-			logger := mocks.NewLogger(ctrl)
-			lcc := mocks.NewLimaCmdCreator(ctrl)
-
 			tc.mockSvc(logger, lcc, ctrl)
-			assert.NoError(t, newRemoveVMAction(lcc, logger).runAdapter(tc.cmd, tc.args))
+			tc.cmd.SetArgs(tc.args)
```

Comment:
Maybe we can

1. Change `cmd *cobra.Command` to `cmd func() *cobra.Command` in the test case definition, and in that func, we can call `cmd.ParseFlags(...)` to set it up if needed.
2. Change what's inside `t.Run` back to the original flow.

Also, probably that's why the test cases cannot be run in parallel now?